ownership-chain/node/src/chain_spec.rs

```diff
@@ -173,7 +173,7 @@ fn testnet_genesis(
 			..Default::default()
 		},
 		balances: laos_ownership_runtime::BalancesConfig {
-			balances: endowed_accounts.iter().cloned().map(|k| (k, 1 << 60)).collect(),
+			balances: endowed_accounts.iter().cloned().map(|k| (k, 1e24 as u128)).collect(),
```

Comment:
why do we need to change initial account in testnets?

Reply:
`1 << 60` amounts to `1152921504606846976` which has length of 19 and is equal to ~1.15 UNIT in the testnet. So when I was testing xcm, it's hard to send whole amounts (i.e if I send 1 UNIT, I will have 0.15 UNIT left). Obviously it's just a matter of decimals, but this was exposed while testing.

I think it is set to `1 << 60` because in `frontier-template` they use 12 decimals and for them it's a huge amount of tokens

Reply:
but it would be probably better to set this relative to the UNIT, i.e 1_000 * UNITS, or sth else